library/lua/gui/widgets.lua

```diff
@@ -456,15 +456,17 @@ function List:init(info)
 end
 
 function List:setChoices(choices, selected)
-    self.choices = choices or {}
+    self.choices = {}
 
-    for i,v in ipairs(self.choices) do
+    for i,v in ipairs(choices) do
```

Comment:
This breaks the ability to do `setChoices()` to clear the list. I don't know if that's important (`setChoices({})` still works), but maybe `choices or {}` here would be better.

Reply:
Thanks. Fixed my error.